src/mkdocstrings/inventory.py

```diff
@@ -7,7 +7,8 @@ def get_inventory(autorefs, project="project", version="0.0.0"):
     inv.version = version
 
     for anchor, url in autorefs._url_map.items():
-        o = soi.DataObjStr(name=anchor, domain="py", uri=url, role="", priority="", dispname="")
+        url = url.replace(anchor, "$")
```

Comment:
Minor potential gotcha here -- this `$` shorthand will only be interpreted correctly by Sphinx when it replaces `anchor` at the *very end* of `url`.

If you know that that'll always be the case, then you should be fine.

Reply:
Yes it's not the most robust code right now, we'll to be sure of that indeed :slightly_smiling_face:

Reply:
I'm now replacing only if there's a `#` before the anchor, and the anchor is at the end.